tests/e2e/vmgroups_test.go

```diff
@@ -111,6 +111,9 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
 	log.Printf(out)
 
+	// Remove Config DB
+	adminutils.ConfigRemove(vg.config.EsxHost)
+
```

Comment:
@lipingxue current test code removes all vmgroups in TearDownSuite() which is _always_ called irrespective of pass/fail of the test suite.

A test must remove all artifacts that are created by it. But why remove the config DB? 
After removing all test artifacts is the DB sane, how do we know that it is?

If we remove the config DB (which btw isn't a test artifact) then most likely we are removing bugs as well. If the removal of test artifacts leaves the DB is a corrupt/inconsistent state - whether local or clustered - then shouldn't that be discovered in the testing?

We can't run without the config DB - customers are trying it out and hence it must be in all our tests.